behat.yml

```diff
@@ -0,0 +1,21 @@
+default:
+  suites:
+    global:
+        contexts:
+            - ApplicationContext: #global context for whole application
+                #session:   '@session' #you can inject symfony services to Context object cnstructor here!
```

Comment:
You could do that but how does it add anything to functional tests that scrape application responses via virtual browser driver?

Behat will provide functional / integration tests of web interface (html) and since you want to use AJAX we will use PhantomJS webdriver as virtual browser. Therefore there should be 0 need for interaction with application in ways other than issuing requests through PhantomJS.